src/mpl/src/thread/mpl_thread_argobots.c

```diff
@@ -0,0 +1,24 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ *  (C) 2015 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ */
+
+/* common header includes */
+#include "mpl.h"
+
+MPL_SUPPRESS_OSX_HAS_NO_SYMBOLS_WARNING;
+
+#if defined(MPL_THREAD_PACKAGE_NAME) && (MPL_THREAD_PACKAGE_NAME == MPL_THREAD_PACKAGE_ARGOBOTS)
+/* begin argobots impl */
+
+MPLI_Thread_info_t MPLI_ThreadInfo = { 0 };
+
+void MPL_thread_create(MPL_thread_func_t func, void *data, MPL_thread_id_t * idp, int *errp)
+{
+    /* not used */
```

Comment:
I think it's okay to do this for now, but I suggest to put a TODO item to enable creation of Argobots ULTs inside MPICH. This will be particularly interesting for asynchronous progress.